packages/widget/src/components/amount-selector/amount-selector.ts

```diff
@@ -132,20 +131,20 @@ export class AmountSelector extends BaseComponent {
               class="amountSelectorInput"
               placeholder="0.000"
               @change=${this._onInputAmountChangeHandler}
-              value=${this.amount || ifDefined(this.preselectedAmount)}
+              value=${this.amount ||
+              ifDefined(this.preselectedAmount).toString()}
```

Comment:
Can you rollback lint changes? It will cause a massive amount of conflicts, it should be in separate PR.